daemon/mgr/container_exec.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/alibaba/pouch/pkg/randomid"
 	"github.com/alibaba/pouch/pkg/streams"
 	"github.com/alibaba/pouch/pkg/user"
+	"github.com/docker/docker/daemon/caps"
```

Comment:
Can we replace this package?

Reply:
The package is just there, in vendor , you can change it if you like